### ISSUE TITLE
Capcom vs. SNK Pro - Missing characters portrait during ending

### ISSUE THREAD

**Issue by BloodRaynare** [libretro/flycast#890]
Hello.

Just wanted to report about the graphical bug in CVSPro.

The characters portrait are supposed to be appear during the endings.

![2020-05-14 09_28_49-Greenshot](https://user-images.githubusercontent.com/46045141/81886543-28936400-95c7-11ea-93ec-334048f3e0d8.png)

But, on the emulator, the portraits are missing.

![Capcom vs  SNK - Millennium Fight 2000 Pro (Japan)-200514-082344](https://user-images.githubusercontent.com/46045141/81886697-8de75500-95c7-11ea-88ea-8572f770c008.png)

It happens on both DC and NAOMI versions.

I'm using latest update of Flycast by the way as I writing this (dbf96863b03ff020e32850b5a748904373056f4f). 

And here's my setting for this game:

> reicast_allow_service_buttons = "enabled"
> reicast_alpha_sorting = "per-pixel (accurate)"
> reicast_analog_stick_deadzone = "15%"
> reicast_anisotropic_filtering = "off"
> reicast_boot_to_bios = "disabled"
> reicast_broadcast = "Default"
> reicast_cable_type = "VGA (RGB)"
> reicast_custom_textures = "disabled"
> reicast_delay_frame_swapping = "disabled"
> reicast_digital_triggers = "disabled"
> reicast_div_matching = "auto"
> reicast_dump_textures = "disabled"
> reicast_enable_dsp = "enabled"
> reicast_enable_naomi_15khz_dipswitch = "disabled"
> reicast_enable_purupuru = "enabled"
> reicast_enable_rttb = "enabled"
> reicast_force_wince = "disabled"
> reicast_frame_skipping = "disabled"
> reicast_framerate = "fullspeed"
> reicast_gdrom_fast_loading = "disabled"
> reicast_hle_bios = "disabled"
> reicast_internal_resolution = "640x480"
> reicast_language = "Default"
> reicast_lightgun1_crosshair = "disabled"
> reicast_lightgun2_crosshair = "disabled"
> reicast_lightgun3_crosshair = "disabled"
> reicast_lightgun4_crosshair = "disabled"
> reicast_mipmapping = "enabled"
> reicast_oit_abuffer_size = "512MB"
> reicast_per_content_vmus = "disabled"
> reicast_pvr2_filtering = "disabled"
> reicast_region = "USA"
> reicast_render_to_texture_upscaling = "1x"
> reicast_screen_rotation = "horizontal"
> reicast_show_lightgun_settings = "disabled"
> reicast_show_vmu_screen_settings = "disabled"
> reicast_synchronous_rendering = "enabled"
> reicast_system = "auto"
> reicast_texupscale = "off"
> reicast_texupscale_max_filtered_texture_size = "256"
> reicast_threaded_rendering = "enabled"
> reicast_trigger_deadzone = "0%"
> reicast_vmu1_pixel_off_color = "DEFAULT_OFF 01"
> reicast_vmu1_pixel_on_color = "DEFAULT_ON 00"
> reicast_vmu1_screen_display = "disabled"
> reicast_vmu1_screen_opacity = "100%"
> reicast_vmu1_screen_position = "Upper Left"
> reicast_vmu1_screen_size_mult = "1x"
> reicast_vmu2_pixel_off_color = "DEFAULT_OFF 01"
> reicast_vmu2_pixel_on_color = "DEFAULT_ON 00"
> reicast_vmu2_screen_display = "disabled"
> reicast_vmu2_screen_opacity = "100%"
> reicast_vmu2_screen_position = "Upper Left"
> reicast_vmu2_screen_size_mult = "1x"
> reicast_vmu3_pixel_off_color = "DEFAULT_OFF 01"
> reicast_vmu3_pixel_on_color = "DEFAULT_ON 00"
> reicast_vmu3_screen_display = "disabled"
> reicast_vmu3_screen_opacity = "100%"
> reicast_vmu3_screen_position = "Upper Left"
> reicast_vmu3_screen_size_mult = "1x"
> reicast_vmu4_pixel_off_color = "DEFAULT_OFF 01"
> reicast_vmu4_pixel_on_color = "DEFAULT_ON 00"
> reicast_vmu4_screen_display = "disabled"
> reicast_vmu4_screen_opacity = "100%"
> reicast_vmu4_screen_position = "Upper Left"
> reicast_vmu4_screen_size_mult = "1x"
> reicast_volume_modifier_enable = "enabled"
> reicast_widescreen_cheats = "disabled"
> reicast_widescreen_hack = "disabled"
> 

And just in case, here's the savestate:
https://drive.google.com/open?id=1MECYKTvn_ACzoPuiOnOj8Wc_Y7fpzzRV

Thank you for your attention.

**Comment by barbudreadmon** [comment]
Could you add savestates to your bug report ?

**Comment by BloodRaynare** [comment]
Yes, I also actually created a savestate just in case when I created this report. I saved it right where the ending cutscene starts. Here's the link:
https://drive.google.com/open?id=1MECYKTvn_ACzoPuiOnOj8Wc_Y7fpzzRV

**Comment by flyinghead** [comment]
Thanks for the savestate, very helpful.

This is another depth issue. Applying some extra depth scaling fixes it but causes major issues in other parts of the game so I'm afraid there's no easy solution for now.
